tests/unittest/_torch/auto_deploy/unit/singlegpu/test_ad_build_small_single.py

```diff
@@ -68,6 +68,54 @@
                 "model_kwargs": {"num_hidden_layers": 2},
             },
         ),
+        # small deepseekv3model with world_size 0 (no processes are spawned) + torch-opt
+        (
+            0,
+            {
+                "model": _hf_model_dir_or_hub_id(
+                    f"{llm_models_root()}/DeepSeek-V3",
+                    "deepseek-ai/DeepSeek-V3",
+                ),
+                "runtime": "demollm",
+                "attn_backend": "TritonWithFlattenedInputs",
+                "compile_backend": "torch-simple",
+                "model_kwargs": {
+                    "num_hidden_layers": 5,
+                    "hidden_size": 32,
+                    "max_position_embeddings": 2048,
+                    "intermediate_size": 16,
+                    "moe_intermediate_size": 16,
+                    "n_routed_experts": 16,
+                    "num_attention_heads": 8,
+                    "num_key_value_heads": 4,
+                },
+                "skip_loading_weights": "True",  # TODO: remove this once we fix fp8 weight loading
+            },
+        ),
+        # small deepseekv3 model with world_size 1 (processes are spawned)
+        (
+            1,
+            {
+                "model": _hf_model_dir_or_hub_id(
+                    f"{llm_models_root()}/DeepSeek-V3",
+                    "deepseek-ai/DeepSeek-V3",
+                ),
+                "runtime": "demollm",
+                "attn_backend": "TritonWithFlattenedInputs",
+                "compile_backend": "torch-simple",
+                "model_kwargs": {
+                    "num_hidden_layers": 6,
+                    "hidden_size": 32,
+                    "max_position_embeddings": 2048,
+                    "intermediate_size": 16,
+                    "moe_intermediate_size": 16,
+                    "n_routed_experts": 16,
+                    "num_attention_heads": 8,
+                    "num_key_value_heads": 4,
+                },
+                "skip_loading_weights": "True",  # TODO: remove this once we fix fp8 weight loading
```

Comment:
nit: for this test I would always use `skip_loading_weights` to speed them up:

```suggestion
                "skip_loading_weights": "True",
```